sympy/stats/symbolic_probability.py

```diff
@@ -85,18 +86,33 @@ class Expectation(Expr):
     >>> from sympy.abc import a
     >>> Expectation(a*X)
     Expectation(a*X)
-    >>> Y = Normal("Y", 0, 1)
+    >>> Y = Normal("Y", 1, 2)
     >>> Expectation(X + Y)
     Expectation(X + Y)
 
-    To expand the ``Expectation`` into its expression, use ``doit()``:
+    To expand the ``Expectation`` into its expression, use ``expand()``:
 
-    >>> Expectation(X + Y).doit()
+    >>> Expectation(X + Y).expand()
     Expectation(X) + Expectation(Y)
-    >>> Expectation(a*X + Y).doit()
+    >>> Expectation(a*X + Y).expand()
```

Comment:
I would also add an example with `Expectation((X + Y)*(X - Y)).expand()`, to show that `.expand()` is recursively called downwards.

Reply:
Also show how `.expand()` acts on multiplication.

Reply:
Yes, added

Reply:
Yes, added!